### PR TITLE
Metric Refresh 

### PR DESCRIPTION
**Pull request by bryce-b**
This PR adds the updated metric spec to API and SDK packages, and marks the existing interface as deprecated.
This is a work in progress

### CODE REVIEW

**Comment by nachoBonafonte** [comment]
Except for the swift-tools-version change mentioned yesterday, everything LGTM

File: Sources/OpenTelemetryApi/Metrics/Stable/Instruments/ObservableResult.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+// Created by Bryce Buchanan on 1/20/22.
```
Comment (bryce-b) [inline Sources/OpenTelemetryApi/Metrics/Stable/Instruments/ObservableResult.swift:2]: correct this

File: Sources/OpenTelemetryApi/Metrics/Stable/Instruments/ObservableMeter.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//
+// Created by Bryce Buchanan on 1/20/22.
+//
```
Comment (bryce-b) [inline Sources/OpenTelemetryApi/Metrics/Stable/Instruments/ObservableMeter.swift:3]: fix this

File: Sources/OpenTelemetryApi/Metrics/Stable/Instruments/Sum/StableCounterMeasurement.swift [hunk 3]
```diff
@@ -0,0 +1,10 @@
+//
+// Created by Bryce Buchanan on 1/28/22.
+//
```
Comment (bryce-b) [inline Sources/OpenTelemetryApi/Metrics/Stable/Instruments/Sum/StableCounterMeasurement.swift:3]: fix this

**Pull request closed without merging**